client/blocks/jetpack-benefits/index.jsx

```diff
@@ -0,0 +1,190 @@
+import QueryJetpackScan from 'calypso/components/data/query-jetpack-scan';
+import { JetpackBenefitsCard } from 'calypso/blocks/jetpack-benefits/benefit-card';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+// named export for cleaner testing of class methods
+export class JetpackBenefits extends React.Component {
+	siteHasBackups() {
+		return 'unavailable' !== this.props.rewindState?.state;
+	}
+
+	siteHasScan() {
+		return 'unavailable' !== this.props.scanState?.state;
+	}
+
+	productHasBackups = ( productSlug ) => {
+		return (
+			// standalone backup product
+			isJetpackBackupSlug( productSlug ) ||
+			// check plans for Jetpack backup features
+			( isJetpackPlanSlug( productSlug ) &&
+				( planHasFeature( productSlug, FEATURE_JETPACK_BACKUP_DAILY ) ||
+					planHasFeature( productSlug, FEATURE_JETPACK_BACKUP_DAILY_MONTHLY ) ||
+					planHasFeature( productSlug, FEATURE_JETPACK_BACKUP_REALTIME ) ||
+					planHasFeature( productSlug, FEATURE_JETPACK_BACKUP_REALTIME_MONTHLY ) ) )
+		);
+	};
+
+	productHasScan( productSlug ) {
+		return (
+			// standalone scan product
+			isJetpackScanSlug( productSlug ) ||
+			// check plans for Jetpack scan features
+			( isJetpackPlanSlug( productSlug ) &&
+				( planHasFeature( productSlug, FEATURE_JETPACK_SCAN_DAILY ) ||
+					planHasFeature( productSlug, FEATURE_JETPACK_SCAN_DAILY_MONTHLY ) ) )
+		);
+	}
+
+	productHasSearch( productSlug ) {
+		return (
+			// standalone search product
+			// there is not currently a isJetpackSearchSlug
+			JETPACK_SEARCH_PRODUCTS.includes( productSlug ) ||
+			// check plans for Jetpack search features
+			( isJetpackPlanSlug( productSlug ) &&
+				( planHasFeature( productSlug, FEATURE_JETPACK_SEARCH ) ||
+					planHasFeature( productSlug, FEATURE_JETPACK_SEARCH_MONTHLY ) ||
+					// This is a bit obscure - checks specifically for Jetpack Business (Professional)
+					// Is it an error that the plan spec in plans-list.js does not contain search features?
+					planHasFeature( productSlug, FEATURE_ALL_PREMIUM_FEATURES_JETPACK ) ) )
+		);
+	}
+
+	productHasAntiSpam( productSlug ) {
+		// check that this product is standalone anti-spam or one of the plans that contains it
+		return (
+			// standalone anti-spam product
+			isJetpackAntiSpamSlug( productSlug ) ||
+			// check plans for anti-spam features
+			( isJetpackPlanSlug( productSlug ) &&
+				( planHasFeature( productSlug, FEATURE_JETPACK_ANTI_SPAM ) ||
+					planHasFeature( productSlug, FEATURE_JETPACK_ANTI_SPAM_MONTHLY ) ) )
+		);
+	}
+
+	productHasActivityLog( productSlug ) {
+		return isJetpackPlanSlug( productSlug ) || isJetpackBackupSlug( productSlug );
+	}
+
+	renderSiteSearch() {
+		const { translate } = this.props;
+		return (
+			<JetpackBenefitsCard
+				headline={ translate( 'Search' ) }
+				description={ translate(
+					'Jetpack Search helps your visitors instantly find the right content – right when they need it.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Incredibly powerful and customizable, Jetpack Search, helps your visitors instantly find the right content – right when they need it.' )` *ES Score: 8*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/improve-jetpack-cancellation-flow)